### PR TITLE
Add pubsub labels to monitor tooling jobs

### PR DESCRIPTION
**Pull request by srinivashegde86**
Part of https://github.com/knative/test-infra/issues/1403

/cc @chaodaiG

### CODE REVIEW

**Comment by chaodaiG** [comment]
/lgtm
/approve
/hold
@Fredy-Z , do we need monitoring also cover issue tracker jobs and perf clusters related jobs?

**Comment by Fredy-Z** [comment]
> /lgtm
> /approve
> /hold
> @Fredy-Z , do we need monitoring also cover issue tracker jobs and perf clusters related jobs?

It would be useful to also have them being monitored. 
Thanks @srinivashegde86 :-)

**Comment by srinivashegde86** [comment]
I moved the issue tracker jobs into its own file. This makes it easier to read and understand.

File: ci/prow/issue_tracker_config.go [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+func (r repoIssue) generateJob(jobName, labelFilter, updatedTime, comment string) {
+	var data periodicJobTemplateData
+	data.Base = newbaseProwJobTemplateData("knative/test-infra")
+	data.Base.ExtraRefs = append(data.Base.ExtraRefs, "  base_ref: "+data.Base.RepoBranch)
+	data.Base.Image = githubCommenterDockerImage
+	data.PeriodicJobName = jobName
+	data.CronString = periodicCron
+	data.Base.Command = jobCmd
+
+	// TODO(Fredy-Z): replace "r.name:knative/test-infra" with "org:knative" after syncing up with the WGs.
```
Comment (adrcunha) [inline ci/prow/issue_tracker_config.go]:
```suggestion
```

File: ci/prow/issue_tracker_config.go [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+        %s`, r.name, labelFilter),
+		"--updated=" + updatedTime,
+		"--token=/etc/housekeeping-github-token/token",
+		comment,
+		"--template",
+		"--ceiling=10",
+		"--confirm",
+	}
+	addVolumeToJob(&data.Base, "/etc/housekeeping-github-token", "housekeeping-github-token", true, "")
+	addMonitoringPubsubLabelsToJob(&data.Base, data.PeriodicJobName)
```
Comment (adrcunha) [inline ci/prow/issue_tracker_config.go]: Do we really need to monitor the issue tracking jobs?

File: ci/prow/issue_tracker_config.go (hunk 1 above)
Comment (Fredy-Z) [inline ci/prow/issue_tracker_config.go]: Can you help remove this comment? I forgot to remove it the last time I updated the code.

File: ci/prow/issue_tracker_config.go [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+		"        %s\\n\\n\n"+
+		"        /lifecycle rotten", r.daysToRot, r.daysToClose, feedbackNote)
+	r.generateJob(jobName, filter, updatedTime, comment)
+
+	jobName = fmt.Sprintf("ci-%s-issue-tracker-close", repoForJob)
+	// Do not look at issues that has frozen label. Only look at rotten labels
+	filter = `
+		-label:lifecycle/frozen
+		-label:lifecycle/stale
+        label:lifecycle/rotten`
```
Comment (Fredy-Z) [inline ci/prow/issue_tracker_config.go]: The indent here seems not right.

File: ci/prow/issue_tracker_config.go (hunk 2 above)
Comment (srinivashegde86) [reply ci/prow/issue_tracker_config.go]: I am not sure. @Fredy-Z  told me to add them :)

Comment (srinivashegde86) [reply ci/prow/issue_tracker_config.go]: removed the issue tracker labels

**Comment by Fredy-Z** [comment]
/hold cancel
/lgtm
Thanks @srinivashegde86 !

**Pull request merged**